_episodes/markdown.md

```diff
@@ -187,34 +187,23 @@ Syntax highlighting of code blocks can increase readability a lot.
 You get nice coloured code by adding an language identifier right after the
 introductory triple backticks.
 
+Let's take an example with Python:
+
+~~~
+```python
+print("Hello everyone!")
+```
+~~~
+
+The code above will produce the following view:
+
+```python
+print("Hello everyone!")
+```
+
 > ## Syntax Highlighting
 >
-> Below are code snippets written in HTML, Python, and R.
-> Copy this code
->
-> ~~~
-> ```_
-> print(paste("How","are","you?"), quote = FALSE)
-> ```
->
-> ```______
-> s = "How are you?"
-> print s
-> ```
->
-> ```____
-> <!DOCTYPE html>
-> <html>
-> <body>
-> <a href=https://carpentries.org/>This is a link</a>
-> </body>
-> </html>
-> ```
-> ~~~
-> {: .source }
->
-> then guess in which programming language each snippet is written,
-> and fill the gaps to activate syntax highlighting, creating this rendered view:
+> Try to reproduce the markdown source code to create the following view:
```

Comment:
Image in the challenge has the python "hello everyone" code block on top from the example before the exercises in it as well as the other python/r/html code blocks.

Reply:
Well cpotted!!! I jux fix it. Thanks